Soccar [Library]/src/nl/soccar/exception/RogueGameServerException.java

```diff
@@ -0,0 +1,24 @@
+package nl.soccar.exception;
+
+/**
+ * An RogueGameServerException is an Exception that is thrown when a connected
+ * Game Server does not return the correct result of the predefined ping
+ * formula.
+ *
+ * @author PTS34A
+ */
+public class RogueGameServerException extends SoccarException {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this class to remove "Exception" or correct its inheritance. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2166)